sql/moz-fx-data-shared-prod/firefox_ios/retention/view.sql

```diff
@@ -0,0 +1,18 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_ios.retention`
+AS
+SELECT
+  *,
+  CASE
+    WHEN first_seen_date = metric_date
+      THEN 'new_client'
```

Comment:
Let's use the same terminology as [desktop](https://github.com/mozilla/bigquery-etl/blob/bf44863a58f5474b6ea4c465ec5d59238febeed3/sql/moz-fx-data-shared-prod/telemetry/desktop_engagement_agg/view.sql#L7-L14): new_profile, repeat_user, existing_user